Azure/sonic-buildimage
# [tacacs]: do not modify local user if there is no priviledge

**- Why I did it**
fix #6617 

[tacacs]: do not modify local user if there is no priviledge
    
    check the uid before modify local user account.
    
    when run sudo, the process the invoke nss_tacplus library
    does not have priviledge to modify the user profile, and
    will generate below error messages:
    
        user_rw@sonic:~$ sudo bash
        usermod: Permission denied.
        usermod: cannot lock /etc/passwd; try again later.
        usermod: Permission denied.
        usermod: cannot lock /etc/passwd; try again later.

**- How I did it**
check uid first, and skip modify local user if uid is not root

**- How to verify it**
manually verify run sudo bash.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
